src/styles/blocks/_button.scss

```diff
@@ -0,0 +1,37 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+  border: 2px solid $colorBlue;
+  border-radius: 8px;
+  text-decoration: none;
+  color: $colorBlue;
+  font-size: 16px;
+  line-height: 14px;
+  font-weight: bold;
+  cursor: pointer;
+  transition: all 0.3s;
```

Comment:
Always add a transition for styles that you want smoothly change
And create transition-duration variable